### PR TITLE
Dev 1

### PR DESCRIPTION
**Pull request by akselatom**

### CODE REVIEW

File: DEV_1/UniqueSymbolsFinder.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    }
+                    
+                    uniqueSymbolsSequence += this.inputString[i - 1];
+                    uniqueSymbolsSequence += this.inputString[i] + " ";
+                }
+                else
+                {
+                    ////clear a string after coincidence of characters
+                    moreThanTwoLettersSequence.Clear();
+
```
Comment (AlexeiRomankov) [inline DEV_1/UniqueSymbolsFinder.cs]: please delete blank line

File: DEV_1/Dev1.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                }
+                else
+                {
+                    Console.WriteLine("String is too short Length<2");
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                throw;
```
Comment (AlexeiRomankov) [inline DEV_1/Dev1.cs:36]: is it needed?

File: DEV_1/Dev1.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        /// <summary>
+        /// The entry point of the program
+        /// </summary>
+        /// <param name="args">Entry string.Specified in the input argument string</param>
+        public static void Main(string[] args)
+        {
+            try
+            {
+                var testClass = new UniqueSymbolsFinder(args[0]);
```
Comment (AlexeiRomankov) [inline DEV_1/Dev1.cs]: if there is no arguments at all?

File: DEV_1/UniqueSymbolsFinder.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            this.inputString = inputString;
+        }
+
+        /// <summary>
+        /// Check string for length
+        /// </summary>
+        /// <returns>Returns true if string length more than 1</returns>
+        public bool CheckStringLength()
+        {
+            return this.inputString.Length >= 2;
```
Comment (AlexeiRomankov) [inline DEV_1/UniqueSymbolsFinder.cs]: Why should "...Finder" "Check" string for extra conditions? Please use Single Responsibility

File: DEV_1/UniqueSymbolsFinder.cs [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        /// </summary>
+        /// <returns>Returns a string in which unique characters are separated by a space.</returns>
+        public string GetUniqueSymbolsSequence()
+        {
+            var uniqueSymbolsSequence = string.Empty;
+            var moreThanTwoLettersSequence = new StringBuilder(string.Empty);
+
+            for (var i = 1; i < this.inputString.Length; i++)
+            {
+
```
Comment (AlexeiRomankov) [inline DEV_1/UniqueSymbolsFinder.cs]: please delete blank line

File: DEV_1/UniqueSymbolsFinder.cs (hunk 1 above)
Comment (akselatom) [reply DEV_1/UniqueSymbolsFinder.cs]: Please confirm the changes for the possibility of merge to master

**Pull request merged**